templates/s3-bucket.yaml

```diff
@@ -0,0 +1,94 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
@philerooski Do you know Tess' reason to create a custom template over using a IT maintained one: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.0/templates/S3/synapse-external-bucket.j2

Reply:
I don't know.